Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -350,25 +350,27 @@ public final class PackageInfoMapper: PackageInfoMapping {
             }
         )
 
-        let targets = try packageInfo.targets.compactMap { target -> ProjectDescription.Target? in
-            guard let products = targetToProducts[target.name] else { return nil }
-            return try Target.from(
-                target: target,
-                products: products,
-                packageName: name,
-                packageInfo: packageInfo,
-                packageInfos: packageInfos,
-                packageFolder: path,
-                packageToProject: packageToProject,
-                productTypes: productTypes,
-                baseSettings: baseSettings,
-                targetSettings: targetSettings,
-                platforms: targetToPlatform,
-                minDeploymentTargets: minDeploymentTargets,
-                targetToResolvedDependencies: targetToResolvedDependencies,
-                targetToModuleMap: targetToModuleMap
-            )
-        }
+        let targets: [ProjectDescription.Target] = try packageInfo.targets
+            .flatMap { target -> [ProjectDescription.Target] in
+                guard let products = targetToProducts[target.name] else { return [] }
+
+                return try [ProjectDescription.Target].from(
```

Comment:
I would rather do a platforms.map { Target.from(…) } to keep the from method simpler

Reply:
I think I misunderstood that one - sorry - I will change the `from` method to keep it simpler

Reply:
Simplified the `from` method